students/elaine_x/session03/list_lab_ex.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+'''
+##########################
+#Python 210
+#Session 03 - List Lab
+#Elaine Xu
+#Jan 28,2019
+###########################
+'''
+
+############# Series 1 ################################
+print("Series 1\nList of Fruit:")
+FRUITS = ['Apples', 'Pears', 'Oranges', 'Peaches']
```

Comment:
since this list is changing, you don't want to capitalize it, as that signifies that its a constant (i.e. immutable or unchanging)

Reply:
Thank you mike for the comments. they are very helpful. When you are done reviewing, should I close this request and re-submit.

Reply:
nope, any changes you make automatically get pulled into the request.  But i've merged this PR anyway so you can edit for the next one if you like.